src/store/ui.ts

```diff
@@ -18,6 +21,14 @@ const uiSlice = createSlice({
         mode: ActiveTool.Add,
         muted: false,
         perspective: false,
+        mobileMode: false,
+        placedPreview: false,
+        previewBlockIds: {
```

Comment:
not sure if `previewBlockIds` should be a part of the state

Reply:
Should I then remove it from the reducer-ui and put it as an object in Face.tsx?